x-pack/plugins/lists/public/lists/mocks/query_wrapper.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+export const createQueryWrapper = (): {
```

Comment:
Nit: I would personally prefer that there be something more than just the file location to indicate that this is a test-specific utility. Even seeing it used in tests doesn't necessarily mean it's not also meant to be production code:

```suggestion
export const createQueryWrapperMock = (): {
```